build/WDQS-frontend/README.md

```diff
@@ -8,6 +8,6 @@ Frontend for the Wikibase Query Service, as seen at [https://query.wikidata.org]
 | --- | --- | --- |
 | `LANGUAGE` | "en" | Language to use in the UI |
 | `BRAND_TITLE` | "DockerWikibaseQueryService" | Name to display on the UI |
-| `WIKIBASE_HOST` | "wikibase.svc" | Hostname of the Wikibase host |
-| `WDQS_HOST` | "wdqs-proxy.svc" | Hostname of the WDQS host (probably READONLY, hence use of the wdqs-proxy service) |
+| `WIKIBASE_HOST` | "wikibase" | Hostname of the Wikibase host (required) |
+| `WDQS_HOST` | "wdqs" | Hostname of the WDQS host (required). IT probably READONLY, hence use of the wdqs-proxy service) |
```

Comment:
>  IT probably READONLY, hence use of the wdqs-proxy service)

I do not understand what this wants to say.

Reply:
Me neither. This is in the diff due to change of the default value, but the docs/txt was not update. This should be reviewed and updated along with the more comprehensive documentation update and review.